tests/workflows/stream_subworkflow_node/workflow.py

```diff
@@ -8,12 +8,12 @@
 from vellum.workflows.state import BaseState
 
 
-class Inputs(BaseInputs):
+class InnerInputs(BaseInputs):
```

Comment:
We can no longer (and should never anyway) share inputs classes across workflows